projects/mirza-common-haskell/src/Mirza/Common/GS1BeamOrphans.hs

```diff
@@ -263,8 +263,8 @@ instance FromField EPC.GS1CompanyPrefix where
 instance ToField EPC.GS1CompanyPrefix where
   toField (EPC.GS1CompanyPrefix pfx) = toField pfx
 
-gs1CompanyPrefixType :: BMigrate.DataType PgDataTypeSyntax EPC.GS1CompanyPrefix
-gs1CompanyPrefixType = textType
+gs1CompanyPrefixFieldType :: BMigrate.DataType PgDataTypeSyntax EPC.GS1CompanyPrefix
```

Comment:
Wondering why the change in name here?

Reply:
I wanted to make it clear that this was a DB type specifically.